reviewresponse.cls

```diff
@@ -94,7 +95,7 @@
 
 %%% Blocks %%%
 \newenvironment{generalcomment}{%
-	\begin{tcolorbox}[attach title to upper, title={General Comments}, after title={.\enskip}, fonttitle={\bfseries}, coltitle={colorcommentfg}, colback={colorcommentbg}, colframe={colorcommentframe},]
+	\begin{tcolorbox}[attach title to upper, title={General Comments}, after title={.\enskip}, fonttitle={\bfseries}, coltitle={colorconcluderesponsefg}, colback={colorcommentbg}, colframe={colorcommentframe},]
```

Comment:
If I see this correctly, the `generalcomment` box now uses the color of the conclude response box